feed/tests/test_comment_views.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+from feed.models import Post, Comment
+from django.contrib.auth.models import User
+
+
+USERNAME1 = 'username1'
+USERNAME2 = 'username2'
+USER_PASS = 'secret_pass'
+POST_TITLE = 'post title'
+POST_CONTENT = 'some post content'
+COMMENT_CONTENT = 'some comment content'
+LOGIN_URL = '/login/'
+REDIRECT_URL_STATUS = 302
+PAGE_NOT_FOUND_STATUS = 404
+
+
+def new_comment_url(post_parent_id):
```

Comment:
Do you use it some where?

Reply:
No, slipped in from another PR, now this function have merged and is part of the main branch.
So it is removed from this PR